apps/consulting/posts/openai-pseudocode.md

```diff
@@ -0,0 +1,274 @@
+---
+title: 'Exploring the Impact of Feedback Loops on LLM Code Generation'
+published: March 27, 2023
+author: chris-ostrouchov
+description: 'As an experiment we wanted to add a feedback loop into LLM generated code via type annotations, docstrings, automated tests, and user feedback. We developed a simple package pseudoscode which uses the OpenAI API for generating code.'
```

Comment:
This should be set off to show it's a name. It'll be rendered plaintext, so backticks won't get code formatting. I suggest double quotes.

```suggestion
description: 'As an experiment we wanted to add a feedback loop into LLM generated code via type annotations, docstrings, automated tests, and user feedback. We developed a simple package, "pseudoscode," which uses the OpenAI API for generating code.'
```

Reply:
I thought we used italics in place of backticks during the migration.